nilearn/connectome/__init__.py

```diff
@@ -4,12 +4,13 @@
 of Gaussian graphical models.
 """
 
-from .connectivity_matrices import (sym_to_vec, ConnectivityMeasure,
-                                    cov_to_corr, prec_to_partial)
+from .connectivity_matrices import (sym_matrix_to_vec, vec_to_sym_matrix,
```

Comment:
I think you should put back sym_to_vec to initialize from nilearn.connectome. Since you are deprecating the function and user cannot initialize function directly from nilearn.connectome rather it should be imported from nilearn.connectome.connectivity_matrices. Am I right ?